infrastructure/authentication/auth_service_impl.go

```diff
@@ -151,6 +151,10 @@ func getPrioritizedApiUrl(customUrl string, engineUrl string) string {
 	defaultUrl := config.DefaultSnykApiUrl
 	customUrl = strings.TrimRight(customUrl, "/ ")
 
+	if customUrl == "https://api.pre-prod.snyk.io" && engineUrl != "" {
+		return engineUrl
+	}
+
```

Comment:
this was only used for testing, right?
In that case, please delete it.

Reply:
Yes, now removed.